modules/end-to-end/pages/building-tekton-tasks.adoc

```diff
@@ -0,0 +1,298 @@
+= Building Tekton tasks as bundles in {ProductName}
+
+This document provides a step-by-step guide on how to onboard your Tekton tasks to {ProductName}, enabling their integration into your build pipelines.
+
+
+== Prerequisites
+
+Before you begin, ensure you have the following:
+
+* A Github repository containing your Tekton tasks with a catalog structure, containing all the additional files (README, TROUBLESHOOTING, etc…)
+
+----
+tasks/
+└── echo/
+    ├── 0.1/
+    │   ├── echo.yaml
+    │   ├── README.md
+    │   ├── USAGE.MD
+    │   └── TROUBLESHOOT.md
+    └── 0.2/
+        ├── echo.yaml
+        └── README.md
+----
+
+* {ProductName} Github application installed in the repository xref:installing:github-app.adoc[GitHub App]
```

Comment:
The installing page is for platform engineers but this page is for users. We might not have a good link for this.

Reply:
Looking at the links, they seem to work fine and the link takes us to the correct location:
https://konflux-ci.dev/docs/installing/github-app/

Reply:
Right, the link is a valid link, I just wasn't sure if it was an appropriate link. :)